### DISCUSSION TITLE
specify training epochs for xtts training recipe

### DISCUSSION THREAD

**Discussion by AshkanArabim** [coqui-ai/TTS#3675] [General Q&A] [answered]
I'm trying to fine-tune XTTSv2 using this [this recipe](https://github.com/coqui-ai/TTS/blob/dev/recipes/ljspeech/xtts_v2/train_gpt_xtts.py) and I'm following [this guide](https://docs.coqui.ai/en/latest/finetuning.html). But the recipe doesn't have a place for me to adjust the number of epochs it trains. It just defaults to 1000. 

What's more interesting is that other recipes, such as [the one for GlowTTS](https://github.com/coqui-ai/TTS/blob/dev/recipes/ljspeech/glow_tts/train_glowtts.py) and [this one for for SpeedySpeech](https://github.com/coqui-ai/TTS/blob/dev/recipes/ljspeech/speedy_speech/train_speedy_speech.py) both have an epoch adjustment. Also notice that all these recipes are for LJSpeech. 

Is there a way for me to adjust the epoch size? Because if not, I'm going to have to keep resuming my training from where the last training session left off.

**Comment by eginhard** [comment, marked as answer]
You can add a line in the `GPTTrainerConfig` in that recipe with a custom value for `epochs`

**Reply by AshkanArabim** [reply]
Yep, that worked. Thanks!
